netebpfext/net_ebpf_ext_hook_provider.c

```diff
@@ -304,7 +303,7 @@ net_ebpf_extension_hook_check_attach_parameter(
  * @retval STATUS_INVALID_PARAMETER One or more arguments are incorrect.
  */
 NTSTATUS
-net_ebpf_extension_hook_provider_attach_client(
+_net_ebpf_extension_hook_provider_attach_client(
```

Comment:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md

> The single underscore (_ ) is reserved for local definitions (static, file-scope definitions). e.g., static ebpf_result_t _do_something(..).

So if this is file local, please add the "static" keyword.  Same for any other functions with a `_` at the start.